pkg/collector/corechecks/cluster/orchestrator/processors/common/base.go

```diff
@@ -53,3 +54,19 @@ func ExtractModelManifests(ctx processors.ProcessorContext, resourceManifests []
 	}
 	return cm
 }
+
+// K8sAgentInfo returns the agent info for k8s
+func K8sAgentInfo() *model.K8SAgentInfo {
+	return &model.K8SAgentInfo{
+		Version:              version.AgentVersion,
+		InstallMethod:        version.InstallMethod,
+		InstallMethodVersion: version.InstallMethodVersion,
```

Comment:
Why merge the install method into the version package instead of directly using `installInfo` here ?

Reply:
Thanks for the suggestion, you are right this is better!
What do you think of this approach? https://github.com/DataDog/datadog-agent/compare/90d5449ef721612c80f3bb99f5be9e85a86f6b20..06977197f2508577ea06e10902dacf2a38e284c6

Reply:
It seems that this approach does not work because of build constraints in the common module, so I moved the InstallInfo variable in our orchestrator config module : 
https://github.com/DataDog/datadog-agent/compare/06977197f2508577ea06e10902dacf2a38e284c6..eeb0b943f8fe72d4bfc8a9f5dd0f8bc83738630d